Lib/idlelib/calltip_w.py

```diff
@@ -4,18 +4,26 @@
 Used by calltip.py.
 """
 from tkinter import Label, LEFT, SOLID, TclError
+from tkinter.scrolledtext import ScrolledText
 
 from idlelib.tooltip import TooltipBase
 
 HIDE_EVENT = "<<calltipwindow-hide>>"
-HIDE_SEQUENCES = ("<Key-Escape>", "<FocusOut>")
+HIDE_SEQUENCES = ("<Key-Escape>",)
 CHECKHIDE_EVENT = "<<calltipwindow-checkhide>>"
 CHECKHIDE_SEQUENCES = ("<KeyRelease>", "<ButtonRelease>")
 CHECKHIDE_TIME = 100  # milliseconds
 
 MARK_RIGHT = "calltipwindowregion_right"
 
 
+def widget_size(widget):
```

Comment:
Can you make this function private please? (that is add an underscore to its name)